sdk/inc/azure/core/az_json.h

```diff
@@ -96,102 +95,111 @@ typedef struct
  * @brief Copies the content of the \p token #az_json_token to the \p destination #az_span.
  *
  * @param[in] json_token A pointer to an #az_json_token instance containing the JSON text to copy to
- * the destination.
- * @param[in] destination The #az_span whose bytes will be replaced by the JSON text from the \p
- * token.
+ * the \p destination.
+ * @param destination The #az_span whose bytes will be replaced by the JSON text from the \p
+ * json_token.
+ *
  * @return An #az_span that is a slice of the \p destination #az_span (i.e. the remainder) after the
  * token bytes have been copied.
  *
  * @remarks The method assumes that the \p destination has a large enough size to hold the contents
- * of \p token.
- * @remarks If \p token doesn't contain any text, this function will just return \p destination.
+ * of \p json_token.
+ * @remarks If \p json_token doesn't contain any text, this function will just return \p
+ * destination.
  */
 az_span az_json_token_copy_into_span(az_json_token const* json_token, az_span destination);
 
 /**
- * @brief Returns the JSON token's boolean.
+ * @brief Gets (as an output parameter) the JSON token's boolean.
```

Comment:
I don't think we need to state the `(as an output parameter)`.

```suggestion
 * @brief Gets the JSON token's boolean.
```